common/configuration/asm_leader.go

```diff
@@ -149,6 +155,9 @@ func (l *asmLeader) store(ctx context.Context, ref Ref, value []byte) (*url.URL,
 	_, err := l.client.CreateSecret(ctx, &secretsmanager.CreateSecretInput{
 		Name:         aws.String(ref.String()),
 		SecretString: aws.String(string(value)),
+		Tags: []types.Tag{
+			{Key: aws.String("ftl"), Value: aws.String(ref.Module.Default("_"))},
```

Comment:
Can the value not be empty? `_` is actually a valid module name, so if it can't be empty it should at least be `-` which is not a valid identifier.

Reply:
It can be empty, pushing a fix